### PR TITLE
elasticsearch: added existsType and putMaping

### PR DESCRIPTION
**Pull request by urossmolnik**
Improvement to existing type definition. Added missing methods:
- existsType
- putMapping

### CODE REVIEW

**Comment by zhengbli** [comment]
@CasperSkydt can you take a look?

File: elasticsearch/elasticsearch.d.ts [hunk 1]
```diff
@@ -125,6 +129,17 @@ declare module Elasticsearch {
         body: string | any;
     }
 
+    export interface IndicesPutMappingParams extends GenericParams {
+        timeout?: Date | number;
+        masterTimeout?: Date | number;
+        ignoreUnavailable?: boolean;
+        allowNoIndices?: boolean;
+        expandWildcards?: string;
```
Comment (CasperSkydt) [inline elasticsearch/elasticsearch.d.ts]: Change it so it reflects the only options that exists.. 
expandWildcards?: "open" | "closed" | "none" | "all";

**Comment by CasperSkydt** [comment]
Only have one small comment. Looks good.

File: elasticsearch/elasticsearch.d.ts (hunk 1 above)
Comment (urossmolnik) [inline elasticsearch/elasticsearch.d.ts]: Changed. I changed expandWildcards in `SuggestParams` and `IndicesGetAliasParams` definitions too.

**Pull request merged**